m4_sessions/1708-inning/group_one.md

```diff
@@ -9,17 +9,30 @@
 * History, now, and future of transhumanism
 * Discussion humanism and see how people's feelings and thoughts change after the session.
 
-### Session Outline:
-* Whiteboard spectrum of how people feel about this.
-  - Have everyone go one by one, introduce themself and point out where they are in the spectrum.
-* Introduction to material
-  - History, current
-* small group discussions
-  - After we have groups discuss a main point or two, we have another pulse check on what people thing of transhumanism.
-* Future of transhumanism
-  - What role do we as developers play
-* another small group discussion
-  - last pulse check and see where people are in how they feel about this
+### Outline
+
+#### Intro (15 mins)
+	1) Quick overview on Transhumanism and what we will be doing today.
+	2) Turn and talk. What are your thoughts on the matter?
+	3) Divide group into spectrums (move around room) Joe plays devils advocate, against. Spectrums  = for, against, unsure
+	
+#### History  / Present (30 mins)
+	1) Video / Article
+	2) Participants countoff then do a turn and talk.
+		- What are your thoughts about the video / article?
+	3) Group discussion on video/article
+	4) Another Video / Article
+	
+#### Dev Role (10 mins)
+	1) Whiteboard Neg / Pos of bio hacking / transhumanism
+	2) We curate an example of both for the group with video/article
+	
+#### Future (10 mins)
+	1) Group questions. Where do you see this heading? What kinds of tech do you see we implement? What types of entertainment do you these issues showing up in that have influenced your thoughts about transhumanism?
```

Comment:
Cool.  I like this...and maybe even saying...how might this impact your future career?